build.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+# stop on most errors
+set -e
+
+RELEASE_PLATFORMS=(byt cht bdw apl cnl icl tgl)
+PLATFORMS=()
+
+PATH=$pwd/local/bin:$PATH
+
+BIN_DIR=$(pwd)
+
+die()
+{
+  >&2 printf '%s ERROR: ' "$0"
+  # We want die() to be usable exactly like printf
+  # shellcheck disable=SC2059
+  >&2 printf "$@"
+  exit 1
+}
+
+print_usage()
+{
+    cat <<EOF
+Build FW and tplg from SOF repo and cp to this repo
+ENV XTENSA_TOOLS_ROOT is needed for FW XCC build
+usage: build.sh [options] platform(s)
+       -s SOF dir
+       -v release version
+       -m path to MEU tool
+       -a Build all platforms
+EOF
+}
+
+while getopts "s:v:m:a" OPTION; do
+  case "$OPTION" in
+    s) SOF_DIR=$OPTARG ;;
+    v) RELEASE_VERSION=$OPTARG ;;
```

Comment:
Should this be release git tag instead of version ?

Reply:
@lgirdwood Let me check if the sof and sof-bin share the same tag or version things.